sites/cheerpx/src/content/docs/11-guides/File-System-support.md

```diff
@@ -69,6 +69,31 @@ Files in the WebDevice are accessed relative to the current page's URL. For exam
 > [!note] Note
 > It's important to note that this behavior depends on the current page's URL, as it uses a relative path. For more predictable results, it's recommended to use absolute paths when possible.
 
+### Using `application/octet-stream` for WebDevice directories
+
+To handle files from WebDevices properly, set default_type `application/octet-stream`. This ensures that files are treated as binary data. **Only apply this to WebDevice directories**, or it may cause issues like broken images or malfunctioning scripts.
+
+**Nginx configuration example**:
+
+```nginx
+http {
+    # Set default type to application/octet-stream for WebDevice directories
+    default_type application/octet-stream;
```

Comment:
This example should be structured differently. The file should be identical to the default configuration, and inside there must be a `location` section specific for the WebDevice that set the default mime type _and_ remove all mime type definitions by using `types { }`

We should also add a example line that creates the WebDevice to show that the path is the same used in the `location` section